### ISSUE TITLE
Worries on vr Term

### ISSUE THREAD

**Issue by xz-keg** [huangwb/AS-GCN#5]
I try to output the value of the reg_loss term of your model to see what is the difference between vr=0 and vr=0.5 (The results seem quite improved in paper for vr=0.5 than vr=0)

But it turns out that the reg_loss term is around the scale of 10^-18(citeseer/pubmed) 10^-14(cora) 10^-9(reddit) compared with the supervised cross entropy term (which has a scale around 2~4) on test set, for vr=0/0.5, or even vr=100.

I wonder how such a 10^-9~10^-18 term will influence performance quite a lot, is there any explanation?

I guess the gradient will vanish if we add two values with such a large difference in scale, also in my experiments, there is hardly any difference in model performance than random fluctuate between vr=0,0.5 and 100.

**Comment by huangwb** [comment]
Hi, you raise a very good question. But now I am rushing for the coming nips submission. I will come back here soon after the ddl. Just put the flag here and remind me then.

**Comment by huangwb** [comment]
I have re-run this code on my machine. It reads that after 300 epoch on Cora, the reg_loss is 0.5 and validation accuracy becomes around 0.86, if no vr (vr=0); and the reg_loss is 0.03 and accuracy is around 0.87 if using vr=0.5. For other datasets like citeseer and pubmed, the reg_loss has a larger scale than what you have reported. Have you printed the right number of reg_loss?

**Comment by xz-keg** [comment]
I output the value of the "reg" term of the GraphSampleConvolutionReg layer ( I believe in all sorts of AS-GCN models there is only one such layer?).

**Comment by huangwb** [comment]
I suggest you to try this 
"outs = sess.run([model.opt_op, model.loss, model.accuracy, model.reg_loss], feed_dict=feed_dict)"
in the run_pubmed.py, where model.reg_loss will give the expected number
